demisto_sdk/commands/validate/file_validator.py

```diff
@@ -276,7 +276,8 @@ def validate_modified_files(self, modified_files):  # noqa: C901
             modified_files (set): A set of the modified files in the current branch.
         """
         for file_path in modified_files:
-            file_type = find_type(file_path)
+            # in case of moving files- file_path is a tuple in length of 2
```

Comment:
how can you know that it is of length 2?  why not check it? do we have a test for the case it is a tuple?

Reply:
Because when the file was renamed it shows the original name and the new name.
 I don't think it can be anything more than that

Reply:
lets clarify the comment and mention its returned from git diff